pyspeckit/spectrum/units.py

```diff
@@ -417,6 +423,11 @@ def __new__(self, xarr, unit="Hz", frame='rest', frame_offset=0.0,
         else:
             subarr.velocity_convention = velocity_convention
 
+        subarr._equivalencies = equivalencies
+        if center_frequency:
```

Comment:
You can either check for `refX` and `center_frequency` here, then pass just one of them to `find_equivalencies`, or you could pass both on to `find_equivalencies` and then have that code check that only one of them has been specified.